Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -204,7 +204,7 @@ public static ExitValue ProcessCommandLineArguments(string[] args, out List<Dafn
         string baseName = cce.NonNull(Path.GetFileName(dafnyFileNames[dafnyFileNames.Count - 1]));
         var verified = Boogie(baseName, boogiePrograms, programId, out statss, out oc);
         var compiled = Compile(dafnyFileNames[0], otherFileNames, dafnyProgram, oc, statss, verified);
-        exitValue = verified && (compiled || !DafnyOptions.O.Compile) ? ExitValue.VERIFIED : !verified ? ExitValue.NOT_VERIFIED : ExitValue.COMPILE_ERROR;
+        exitValue = verified && compiled ? ExitValue.VERIFIED : !verified ? ExitValue.NOT_VERIFIED : ExitValue.COMPILE_ERROR;
```

Comment:
I'd suggest breaking this `? :` statement into an if-then-else. Already there are too many issues with this line it seems.